src/api/controllers/misc.ts

```diff
@@ -19,12 +19,12 @@ import { cacheHandlers } from "./cache.ts";
 
 /** This function is the internal handler for the ready event. Users can override this with controllers if desired. */
 export async function handleInternalReady(
-  data: DiscordPayload,
+  data: GatewayPayload,
   shardID: number,
 ) {
   if (data.t !== "READY") return;
 
-  const payload = data.d as ReadyPayload;
+  const payload = data.d as ReadyEventFields;
```

Comment:
Rename `ReadyEventFields` to `ReadyEventPayload` to ensure consistency with other event payload types.